cpp/src/gandiva/engine.h

```diff
@@ -54,6 +54,13 @@ class GANDIVA_EXPORT Engine {
     functions_to_compile_.push_back(fname);
   }
 
+  /// Set BaseObjectCache.
+  template <class KeyType>
+  Status SetLLVMObjectCache(GandivaObjectCache<KeyType>& object_cache) {
+    execution_engine_->setObjectCache(&object_cache);
+    return Status::OK();
```

Comment:
status is not required here. just make it void.

Reply:
Done